monai/transforms/croppad/dictionary.py

```diff
@@ -669,7 +677,11 @@ def __init__(
         self.max_roi_scale = max_roi_scale
 
     def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, NdarrayOrTensor]:
-        img_size = data[first(self.key_iterator(data))].shape[1:]  # type: ignore
+        first_key: Union[Hashable, List] = next(self.key_iterator(data), [])  # type: ignore
+        if first_key == []:
+            return data  # type: ignore
```

Comment:
This one has the type ignore, any idea why it's different from the other modifications?

Reply:
Hi @wyli ,

That because this transform is subclass of another transform and don't have the d = dict(data) logic, d has different type from data.

Thanks.